src/hardware/mixer.cpp

```diff
@@ -368,6 +375,45 @@ void MixerChannel::AddSilence()
 	MIXER_UnlockAudioDevice();
 }
 
+void MixerChannel::EnableLowPassFilter(const bool enabled)
+{
+	filter.enabled = enabled;
+}
+
+void MixerChannel::ForceLowPassFilter(const bool force)
+{
+	filter.force = force;
+}
+
+void MixerChannel::ConfigureLowPassFilter(const uint8_t order,
+                                          const uint16_t cutoff_freq)
+{
+	assert(order > 0 && order <= max_filter_order);
+	const auto sample_rate = mixer.freq;
+	for (auto i = 0; i < 2; ++i)
+		filter.lpf[i].setup(order, sample_rate, cutoff_freq);
```

Comment:
If we can discover the loop extent directly (instead of using the `2`), then that's preferred.

Maybe get the `.size()` or loop over the items themselves:

``` c++
	for (auto & f : filter.lpf)
		f.setup(order, sample_rate, cutoff_freq);
```